Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -1074,13 +1082,20 @@ def ruleset_create_command(client: PolicyComputeEngine, args: Dict[str, Any]) ->
         json_response = response.to_json()
         json_response["already_exists"] = False
         readable_output = prepare_ruleset_create_output(json_response, name)
-    except IllumioException:
-        rule_sets = client.rule_sets.get(params={"name": name})
-        for rule_set in rule_sets:
-            json_response = rule_set.to_json()
-            json_response["already_exists"] = True
-            if json_response.get("name") == name:
-                readable_output = prepare_ruleset_create_output(json_response, name, already_exists=True)
+    except Exception as e:
+        if EXISTING_RULESET in str(e):
+            try:
+                rule_sets = client.rule_sets.get(params={"name": name})
+                for rule_set in rule_sets:
+                    if rule_set.name == name:
+                        json_response = rule_set.to_json()
+                        json_response["already_exists"] = True
+                        readable_output = prepare_ruleset_create_output(json_response, name, already_exists=True)
+                        break
+            except Exception as e:
+                raise Exception("Encountered error while creating rule set: {}".format(e))
+        else:
+            raise Exception("Encountered error while creating Ruleset: {}".format(e))  # type: ignore
```

Comment:
Please make the error messages the same here.

Reply:
Yes, I've made the changes.